sources/AppBundle/Association/Model/Repository/CompanyMemberRepository.php

```diff
@@ -5,14 +5,21 @@
 namespace AppBundle\Association\Model\Repository;
 
 use AppBundle\Association\Model\CompanyMember;
+use Assert\Assertion;
 use Aura\SqlQuery\Common\SelectInterface;
 use CCMBenchmark\Ting\Driver\Mysqli\Serializer\Boolean;
+use CCMBenchmark\Ting\Repository\CollectionInterface;
+use CCMBenchmark\Ting\Repository\HydratorArray;
 use CCMBenchmark\Ting\Repository\HydratorSingleObject;
 use CCMBenchmark\Ting\Repository\Metadata;
 use CCMBenchmark\Ting\Repository\MetadataInitializer;
 use CCMBenchmark\Ting\Repository\Repository;
 use CCMBenchmark\Ting\Serializer\SerializerFactoryInterface;
+use InvalidArgumentException;
 
+/**
+ * @method CompanyMember|null get($primariesKeyValue, $forceMaster = false)
```

Comment:
Cette annotation elle va perturber l'auto-complétion de PHPStorm (il y avait la même dans [UserRepository](https://github.com/afup/web/commit/9667f774d71c3384378417450e9c784ef9fdbe35)).

À la place tu peux faire ça :
```suggestion
 * @extends Repository<CompanyMember>
```